core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -109,6 +109,23 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->always(static fn (string $value): string => Path::canonicalize($value))
                     ->end()
                 ->end()
+                ->arrayNode('allowed_protocols')
```

Comment:
Allowed Protocols is a bit too generic I think.

Maybe add something like url, link, html attribute to the name to make it more clear?

Or how about moving it to its own section `html_filter`?
The allowed attributes and tags can be moved here too then in the future.

Reply:
> Allowed Protocols is a bit too generic I think

Yes, that sounds like Contao would disallow others in general.

Maybe `contao.sanitizer.allowed_url_protocols`?

The problem is, this targets a helper function where the usage or promoted usage is unclear (yet).

Reply:
+1 for `contao.sanitizer.allowed_url_protocols`